examples/web/menu.html

```diff
@@ -0,0 +1,106 @@
+<!-- The software contained in this file is free and unencumbered software
+released into the public domain. Anyone is free to use the software contained
+in this file as they choose, including incorporating it into proprietary
+software.-->
+
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>GSTD HTTP server examples</title>
+</head>
+
+<body>
+
+    <link href="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.0/css/bootstrap.min.css" rel="stylesheet" id="bootstrap-css">
+    <script src="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.0/js/bootstrap.min.js"></script>
+
+    <!------ Include the above in your HEAD tag ---------->
+
+    <link href="http://netdna.bootstrapcdn.com/font-awesome/4.0.3/css/font-awesome.min.css" rel="stylesheet" type="text/css" />
+
+
+    <!-- Image and text -->
+
+    <div style="padding-top: 50px;background-color: rgb(255, 255, 255);width: 65vw;height: 100vh;" class="container">
+
+        <nav style="padding-bottom: 76px;" class="navbar navbar-light" style="background-color: #ffffff;">
+            <a class="navbar-brand" style="font-size: x-large;" href="https://www.ridgerun.com/" target="_blank">
+                <img src="src/resources/logo.png" width="250" height="auto" style="padding-bottom: 15px;" class="d-inline-block align-top" alt=""> GSTD HTTP client examples
+            </a>
+        </nav>
+        <p>
+            <label for="quantity">Address:</label>
+            <input id="address" type="text" minlength="7" maxlength="15" size="15" pattern="^((\d{1,2}|1\d\d|2[0-4]\d|25[0-5])\.){3}(\d{1,2}|1\d\d|2[0-4]\d|25[0-5])$">
+        </p>
+
+        <label for="quantity">Port:</label>
+        <input type="number" id="port">
+
+        <h3>Examples</h3>
+        <div class="container">
+            <div class="row">
+                <div class="col-12">
+                    <table class="table table-bordered">
+                        <thead>
+                            <tr>
+                                <th scope="col">Name</th>
+                                <th scope="col">Description</th>
+                                <th scope="col" style="width: 15px;">Run</th>
+                            </tr>
+                        </thead>
+                        <tbody>
+                            <tr>
+                                <th scope="row"> Video player </th>
+                                <td>Basic video player (camera or file)</td>
```

Comment:
Example showing how to perform video playback with trick modes from a file or a camera.